### ISSUE TITLE
"Add new account" does not work when username already registered for next index

### ISSUE THREAD

**Issue by friedger** [leather-io/extension#626]
I have some usernames registered with my pass phrase. Not all are shown in app.blockstack.org, all are shown in browser.blockstack.org.

When clicking on "Add new account" in app.blockstack.org, registering a new username fails.

![Screenshot from 2020-10-12 17-48-26](https://user-images.githubusercontent.com/1449049/95766150-25ee8500-0cb3-11eb-94e3-6be2d2e205d9.png)

The next index in app.blockstack.org has already a registered username.

app.blockstack.org should try to find existing usernames on the next index.

**Comment by whoabuddy** [comment]
I am having a similar issue, and I'm trying to understand the best way to approach it.

On browser.blockstack.org, I have my main user name plus two additional names, all three ending with `id.blockstack`.

On app.blockstack.org, I can only see the main user name and one additional name. The third one does not show up, nor am I sure how to add it.

**Comment by markmhendrickson** [comment]
@hstove IIRC we should be loading all of the key's usernames upon authentication, not just those created with Connect, correct?

@friedger I believe your screenshot there is unrelated in that you're rate limited. Does it go away if you change networks?

**Comment by friedger** [comment]
@markmhx I tried on mobile with the same result. I think the error message is wrong.

**Comment by aulneau** [comment]
I do remember running into this kind of thing with my IDs, too.

**Comment by whoabuddy** [comment]
I tried creating a new username, entering the one from my 3rd ID, and receive the message: `This username is not available`.

There's also no easy way to get back to the Choose an Account screen after clicking Add a new account. I used `Alt + Left Arrow` but maybe there should be a button?

![Screenshot from 2020-10-14 07-52-05](https://user-images.githubusercontent.com/9038904/96006323-36f1e000-0df2-11eb-97bc-5c51eef0bc25.png)
